docs/docsite/rst/playbooks_filters.rst

```diff
@@ -70,6 +70,11 @@ Jinja2 provides a useful 'default' filter, that is often a better approach to fa
 In the above example, if the variable 'some_variable' is not defined, the value used will be 5, rather than an error
 being raised.
 
+In the case the variable evaluates to a false value (e.g. empty string), the second parameter of the filter must be set
+to `true`:
+
+    {{ '' | default('the string was empty', true) }}
```

Comment:
not sure this is a good explanation of what that parameter is.

Reply:
It's based on the [documentation](https://github.com/pallets/jinja/blob/master/jinja2/filters.py#L355) of the filter itself. But feel free to suggest better wording and I will amend the commit.

Reply:
I have modified the commit a bit to actually show better example. This is actually how I found found it when using ENV vars and not getting the default value if the variable wasn't defined because Ansible set's the undefined value to empty string.